src/validations/rules/ssp.sch

```diff
@@ -2209,6 +2210,67 @@ leveraged-authorization.</sch:assert>
                         test="@value castable as xs:date and xs:date(@value) gt current-date()">Planned completion date is not past.</sch:assert>
         </sch:rule>
     </sch:pattern>
+    <sch:pattern doc:guide-reference="DRAFT Guide to OSCAL-based FedRAMP System Security Plans §4.13-14"
+                 id="cloud-models">
+        <sch:title>Cloud Service and Deployment Models</sch:title>
+        <sch:let name="service-models"
+                 value="$fedramp-values//fedramp:value-set[@name = 'service-model']//fedramp:enum/@value" />
+        <sch:let name="deployment-models"
+                 value="$fedramp-values//fedramp:value-set[@name = 'deployment-model']//fedramp:enum/@value" />
+        <sch:rule context="oscal:system-characteristics">
```

Comment:
NB: Would you prefer the `$fedramp-values//fedramp:value-set/@name`s be `@cloud-service-model` and `@cloud-service-model` respectively? I forgot how this upsets me, personally, but if you do not mind I will not touch it in GSA#152.

Reply:
I am indifferent though changing the names to resemble the property names seems appropriate. It need not be done soon.